docs/explanation/lifecycle-layer.rst

```diff
@@ -0,0 +1,209 @@
+
+From prime step to OCI layer
+============================
+
+Rockcraft is a tool that creates OCI images using the same concepts and
+mechanisms that create snaps and charms: the lifecycle language from
+Craft Parts. There is a significant difference between the way the Craft
```

Comment:
This might be an appropriate place to cross-link to [the craft-parts documentation](https://canonical-craft-parts.readthedocs-hosted.com/en/latest/).